src/libstd/lib.rs

```diff
@@ -271,6 +271,7 @@
 #![feature(vec_push_all)]
 #![feature(zero_one)]
 #![feature(question_mark)]
+#![feature(env_iterators)]
```

Comment:
Ah this isn't actually required for stable features, so it's ok to leave this off